test/integration/test_sagemaker_extension_wrapper.py

```diff
@@ -22,10 +23,14 @@ def test_initialize_sme_extension(
     secrets.save(CKey.aws_access_key_id, "FAKEKEYIDDONTUSEIT")
     secrets.save(CKey.aws_secret_access_key, "FakeSecretAccessKeyDontTryToUseIt")
 
-    # Run the extension deployment.
-    initialize_sme_extension(secrets)
-
     with open_pyexasol_connection(secrets) as pyexasol_connection:
+        # Set pre-existing language definition with the chosen alias.
+        language_alias = 'PYTHON3_SME'
```

Comment:
In the TE test you used language_alias = f"PYTHON3_TE_{test_name.upper()}" does it make sense to do the same here

Reply:
Currently, we only have one test, so it doesn't matter. In the TE wrapper test, we probably borrowed the code from where there were multiple tests.